File: Source/depreciated/news.py
import json
from urllib.request import urlopen

from depreciated import ApiKeys


def news():
    hlns = []
    u = 'https://newsapi.org/v1/articles?source=associated-press&sortBy=top&apiKey=' + ApiKeys.keyFinder('news')
    c = urlopen(u)
    parsed_json = json.loads(c.read().decode('UTF-8'))
    for H in range(0, 8):
        hlns.append(parsed_json['articles'][H]['title'])
    return hlns


#return "$('#newsJQ').text(articles[i]"

File: Source/static/news.js
var currentArticle = -1,
    list=[];

function getNews(){
    $.get(
        "https://newsapi.org/v1/articles?source=bbc-news&apiKey=d74b51d7710b45009dbb8e22bb21f0ec",
        function show(json) {
            console.log("Running");
            for (let x = 0; x < json.articles.length; x++){
                list.push(json.articles[x]);
            }
            setInterval(post, 5000);
        }
    );
}

function post(){
    if (currentArticle < list.length -1){
        currentArticle++
    } else {
        currentArticle = 0;
        getNews();
    }
    $('#newsJQ').text(list[currentArticle].title);
    $('#newsDesc').text(list[currentArticle].description);
    console.log(currentArticle)
}


window.onload= getNews();